Packs/IPWhitelisting/pack_metadata.json

```diff
@@ -1,21 +1,38 @@
 {
-  "name": "IP Whitelisting",
-  "description": "Playbooks to enable access to Cloud and SaaS services by updating whitelists of source IP addresses.",
-  "support": "xsoar",
-  "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
-  "email": "",
-  "created": "2020-04-22T06:05:52Z",
-  "categories": [
-    "Network Security"
-  ],
-  "tags": [
-    "NetworkSecurity",
-    "Whitelist",
-    "Cloud",
-    "PrismaAccess"
-  ],
-  "useCases": [],
-  "keywords": []
+    "name": "IP Whitelisting",
```

Comment:
@liorkol as discussed please don't forget to remove this "IP Whitelisting" pack and sort the content to the relevant packs (okta, aws, gcp) 
@anara123 FYI as well.

Reply:
Removing the pack.